model_compression_toolkit/core/pytorch/back2framework/pytorch_model_builder.py

```diff
@@ -66,31 +66,39 @@ def _build_input_tensors_list(node: BaseNode,
     return input_tensors
 
 
-def _merge_inputs(_node: BaseNode, input_tensors: List, op_call_args: List,
-                  tensor_input_indices: List = None) -> List:
+def _merge_inputs(_node: BaseNode, input_tensors: List, op_call_args: List, op_call_kwargs: Dict,
+                  tensor_input_allocs: List = None) -> List:
     """
     Merge input tensors list with positional weights and op_call_args, according to correct order.
 
     Args:
         _node: The node the inputs are for.
         input_tensors: activation input tensors to node.
         op_call_args: framework node call args.
+        op_call_kwargs: framework node call kwargs.
+        tensor_input_allocs: List of input allocations to node.
 
     Returns:
         Combined list of input_tensors and op_call_args.
     """
-    if isinstance(_node, FunctionalNode) and _node.tensor_input_indices:
+    if isinstance(_node, FunctionalNode) and _node.tensor_input_allocs:
```

Comment:
what happens if _node.tensor_input_allocs is None but the argument tensor_input_allocs is not None? is that case even possible?
if not, is `tensor_input_allocs == _node.tensor_input_allocs` always?

maybe at least explain what is the difference between them

Reply:
I'll fix it in the next refactor PR, when handling the `inputs_as_list` comment.